src/qonlinetranslator.cpp

```diff
@@ -500,6 +501,8 @@ QString QOnlineTranslator::languageName(Language lang)
         return tr("Bengali");
     case Bosnian:
         return tr("Bosnian");
+    case BritishEnglish:
+        return tr("English (British Pronunciation)");
```

Comment:
Should "pronunciation" be lowercase?
```suggestion
        return tr("English (British pronunciation)");
```